hl_pitch.md

```diff
@@ -0,0 +1,28 @@
+# Human learning pitch
+
+## Preparation
+
+Do the path in the application at least once before to see if anything crashes.
+
+Predownload an image if the demo is done on an emulator, then add this image during the demo instead of taking one.
+
+## Script
+
+Verbal communication is central to human interaction. This means that if you can't communicate verbally, you cannot integrate well into society. People with mental disabilities can have problems communicating verbally, which is why they often use picture based communication.
+
+The problem now is that the same people do not have good abstraction capabilites. If they show you a picture of a fork and you give them a fork that does not look like the fork in the picture they do not recognize it as being what they requested.
+
+A method to teach this abstraction is picture matching, which would traditionally require printing lots of images. With our app you can do all of this digitally, which greatly simplifies the process.
+So let's learn!
+
+I have a tablet here with a big screen so I can see the images better. Let's see what datasets are available to me...
+
+_goes and looks at dataset overview_
+
+But there are only very few images... I want to have more objects to match! This is why I am going to go and augment my dataset.
+
+_adds picture and shows it added afterwards_
+
+Now let's learn some abstraction!
+
+_goes and learns in representation mode_
```

Comment:
```suggestion
So let's learn!

Let's see what datasets are available to me...

_goes and looks at datasets overview (In the learning part)_ 

Let's select the representation mode...

_goes in representation mode and start learning and remark that a picture shouldn't be in this dataset_

 This picture shouldn't be in this dataset and I don't have enough pictures to learn well... Let's see what is going on.
 
 _goes in datasets overview, select the good dataset, first add a new picture from the storage, then go check to see if the picture has been added. There is also the picture that should not be in this dataset, delete it._
 
 Nice, now it is perfect ! I can go back to learning with my improved dataset.
```

Reply:
`Let's select the representation mode...` is not necessary imo. We can see that you do it.

Reply:
You're right :)